cleaning_data.py

```diff
@@ -0,0 +1,199 @@
+"""change documents to UTF-8"""
+
+
+"""tohle má ostatní texty uložit do UTF-8 formátu
+Taky by to mělo vymazat všechno, kromě textu knihy ze zdrojů gutenberg"""
+"""CHANGE documents to UTF-8 and save as a new files and convert to one dictionary"""
+import os
+import glob
+
+def read_text_files_from_subfolders(main_folder_path):
+# Find all .txt files in the specified main folder and its subfolders
+    text_files = glob.glob(os.path.join(main_folder_path, '**', '*.txt'), recursive=True)
+    
+    # Dictionary to store the contents of each file
+    file_contents = {}
+    
+    # Loop through the text files, read their content, and store in dictionary
+    for i, file in enumerate(text_files):

```

Comment:
usually, file is used for open file and path for file path
but `file` is function in Python` so even better is something like
```
for i, path in enumerate(text_file_paths):
    with open(path, 'r', encoding='utf-8-sig') as f:
```